encord/objects/ontology_object_instance.py

```diff
@@ -136,6 +144,22 @@ def _last_frame(self) -> Union[int, float]:
         else:
             return self._parent.number_of_frames
 
+    @property
+    def range_list(self) -> Ranges:
+        if self._range_only:
+            return self._range_manager.get_ranges()
+        else:
+            raise LabelRowError(
+                "No ranges available for this classification instance."
```

Comment:
Update this message to say `object instance`, instead of `classification instance`